ZAPD/Globals.cpp

```diff
@@ -1,7 +1,7 @@
 #include "Globals.h"
-#include <algorithm>
 #include <Utils/File.h>
 #include <Utils/Path.h>
```

Comment:
I know this wasn't you, but those includes aren't including a system library, so those should be using `""` instead of `<>`.
Since this is a clean-up PR, would be nice to change those too.